homeassistant/components/xiaomi_aqara/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Xiaomi Gateway (Aqara)",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/xiaomi_aqara",
-  "requirements": ["PyXiaomiGateway==0.13.4"],
+  "requirements": ["PyXiaomiGateway==0.14.1"],
```

Comment:
The library should close the transport in the event loop.

`loop.create_datagram_endpoint` returns a tuple with the transport and protocol. The library is naming the variables in a confusing way.

https://github.com/Danielhiversen/PyXiaomiGateway/blob/d0214c2ba52a9623b898e8c80ebc8d326b830775/xiaomi_gateway/__init__.py#L122-L123

It's not a coroutine. It's the protocol. And why not await `_create_udp_listener` directly?

Reply:
Thank you very much for explaining, I am learning day by day!

I think I am now indeed closing the transport in the event loop.
I made a upstream library PR to clearify the naming here: https://github.com/Danielhiversen/PyXiaomiGateway/pull/197 (but we do not have to wait for that since it does not change the functioning).

That was because I used examples from other libraries that I addepted, I am still learning a lot about async.

Thank you for your help!